--- Running scipy.signal.find_peaks#40YY/find_peaksYY, strategy: relancer
[ERR MSG]: OrderedDict([('project', 'scipy.signal.find_peaks#40YY'), ('notebook', 'find_peaksYY'), ('text', '')])
[INFO] This case is fully fixed!


real	0m0.581s
user	0m2.128s
sys	0m6.232s
